server/integration_tests/test_data/detection_api_statvars/correlatewithgdpofcalifornia/chart_config.json

```diff
@@ -41,7 +41,7 @@
     "dc/topic/Economy",
     "dc/topic/sdg_17.1.1",
     "dc/topic/EconomicEquity",
-    "dc/topic/GDP",
-    "Amount_EconomicActivity_GrossDomesticProduction_RealValue"
+    "Amount_EconomicActivity_GrossDomesticProduction_RealValue",
```

Comment:
why are these two swapped?

Reply:
looks like this is the order on autopush so I don't think it's a change from this PR:

https://screenshot.googleplex.com/5JVV9RjsDTiw2Dv